### PR TITLE
Improve deCONZ switch platform documentation

### PR DESCRIPTION
**Pull request by Kane610**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15896

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.deconz.markdown [hunk 1]
```diff
@@ -15,10 +15,12 @@ ha_iot_class: "Local Push"
 
 See the [deCONZ main component](/components/deconz/) for configuration instructions.
 
-The `Entity Id` names will be `switch.device_name`, where `device_name` is defined in deCONZ.
+Switches are devices like power plugs and sirens.
 
 Note that devices in the switch platform identify as lights, so there is a manually curated list that defines which "lights" are switches.
 
+The `Entity Id` names will be `switch.device_name`, where `device_name` is defined in deCONZ.
```
Comment (MartinHjelmare) [inline source/_components/switch.deconz.markdown:22]: Do we refer to entity_id as `Entity Id`elsewhere in the docs?

Comment (Kane610) [reply source/_components/switch.deconz.markdown:22]: It seems to vary

Comment (frenck) [reply source/_components/switch.deconz.markdown:22]: I would suggest is replacing it with either:

"The entity ID names will" 
OR
"The `entity_id` names will".

I'll guess the latter is more fitting, as it matches the rest of the sentence.

Comment (Kane610) [reply source/_components/switch.deconz.markdown:22]: Then for consistencies sake I should do the same thing in all deconz pages.

Is it ok if I do that in a clean up PR?

Comment (frenck) [reply source/_components/switch.deconz.markdown:22]: Yeah, I guess it won't hurt that much now at this moment.
Looking forward to the PR 👍

**Review by frenck — APPROVED** [review]

File: source/_components/switch.deconz.markdown (hunk 1 above)
Comment (Kane610) [reply source/_components/switch.deconz.markdown:22]: You'll have it during this week

**Pull request merged**